--- cmake/config-ix.cmake.orig
+++ cmake/config-ix.cmake
@@ -264,13 +264,15 @@
   mipsel mips64 mips64el powerpc64 powerpc64le)
 filter_available_targets(SAFESTACK_SUPPORTED_ARCH x86_64 i386 i686)
 
 if(ANDROID)
   set(OS_NAME "Android")
 else()
   set(OS_NAME "${CMAKE_SYSTEM_NAME}")
 endif()
 
+if (OS_NAME MATCHES "FOO" AND
     (OS_NAME MATCHES "Android|Darwin|Linux|FreeBSD" OR
     (OS_NAME MATCHES "Windows" AND MSVC)))
   set(COMPILER_RT_HAS_SANITIZER_COMMON TRUE)
--- ./lib/Makefile.mk.orig
+++ lib/Makefile.mk
@@ -10,10 +10,6 @@
 SubDirs :=
 
 # Add submodules.
-SubDirs += asan
 SubDirs += builtins
 SubDirs += interception
-SubDirs += lsan
 SubDirs += profile
-SubDirs += sanitizer_common
-SubDirs += ubsan
