--- cmake/config-ix.cmake.orig
+++ cmake/config-ix.cmake
@@ -273,7 +273,7 @@
 if (SANITIZER_COMMON_SUPPORTED_ARCH AND NOT LLVM_USE_SANITIZER AND
     (OS_NAME MATCHES "Android|Darwin|Linux|FreeBSD" OR
     (OS_NAME MATCHES "Windows" AND MSVC)))
-  set(COMPILER_RT_HAS_SANITIZER_COMMON TRUE)
+  set(COMPILER_RT_HAS_SANITIZER_COMMON FALSE)
 else()
   set(COMPILER_RT_HAS_SANITIZER_COMMON FALSE)
 endif()
--- ./lib/Makefile.mk.orig
+++ lib/Makefile.mk
@@ -10,10 +10,6 @@
 SubDirs :=
 
 # Add submodules.
-SubDirs += asan
 SubDirs += builtins
 SubDirs += interception
-SubDirs += lsan
 SubDirs += profile
-SubDirs += sanitizer_common
-SubDirs += ubsan
